lib/src/tasks/test/api.dart

```diff
@@ -59,7 +59,13 @@ TestTask test(
   });
 
   stdoutc.stream.listen(task._testOutput.add);
-  process.stderr.listen(task._testOutput.addError);
+  process.stderr.listen((line) async {
+    task._testOutput.addError(line);
+    if (line.contains('No tests match regular expression')) {
+      await SeleniumHelper.killChildrenProcesses();
+      outputProcessed.complete();
+    }
+  });
```

Comment:
The test was expecting an exception that happened when an error came across the stream.  By handling the onErrors, this exception went away.  Shutting down the test process causes a new exception that makes the test pass correctly now.